boards/arc/axs/board.cmake

```diff
@@ -0,0 +1,3 @@
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
Please add the following line here:
```
board_runner_args(openocd "--use-elf")
```

See https://github.com/zephyrproject-rtos/zephyr/pull/22891 for more details.